packages/apidom-reference/test/parse/parsers/workflows-json-1/index.ts

```diff
@@ -0,0 +1,196 @@
+
+            assert.isFalse(await parser.canParse(file));
+          });
+        });
+      });
+
+      context('given file with unknown extension', function () {
+        specify('should return false', async function () {
+          const file = File({
+            uri: '/path/to/workflows.yaml',
+            mediaType: mediaTypes.latest('json'),
+          });
+          const parser = WorkflowsJson1Parser();
+
+          assert.isFalse(await parser.canParse(file));
+        });
+      });
+
+      context('given file with no extension', function () {
+        specify('should return false', async function () {
+          const file = File({
+            uri: '/path/to/workflows',
+            mediaType: mediaTypes.latest('json'),
+          });
+          const parser = WorkflowsJson1Parser();
+
+          assert.isFalse(await parser.canParse(file));
+        });
+      });
+
+      context('given file with supported extension', function () {
+        context('and file data is buffer and can be detected as Workflows 1.0.0', function () {
+          specify('should return true', async function () {
+            const url = path.join(__dirname, 'fixtures', 'sample-workflow.json');
+            const file = File({
+              uri: '/path/to/workflows.json',
+              data: fs.readFileSync(url),
+            });
+            const parser = WorkflowsJson1Parser();
+
+            assert.isTrue(await parser.canParse(file));
+          });
+        });
+
+        context('and file data is string and can be detected as Workflows 1.0.0', function () {
+          specify('should return true', async function () {
+            const url = path.join(__dirname, 'fixtures', 'sample-workflow.json');
+            const file = File({
+              uri: '/path/to/workflows.json',
+              data: fs.readFileSync(url).toString(),
+            });
+            const parser = WorkflowsJson1Parser();
+
+            assert.isTrue(await parser.canParse(file));
+          });
+        });
+      });
+    });
+
+    context('parse', function () {
+      context('given Workflows 1.0.0 JSON data', function () {
+        specify('should return parse result', async function () {
+          const url = path.join(__dirname, 'fixtures', 'sample-workflow.json');
+          const data = fs.readFileSync(url).toString();
+          const file = File({
+            url,
+            data,
+            mediaType: mediaTypes.latest('json'),
+          });
+          const parser = WorkflowsJson1Parser();
+          const parseResult = await parser.parse(file);
+
+          assert.isTrue(isParseResultElement(parseResult));
+        });
+      });
+
+      context('given Workflows 1.0.0 JSON data as buffer', function () {
+        specify('should return parse result', async function () {
+          const url = path.join(__dirname, 'fixtures', 'sample-workflow.json');
+          const data = fs.readFileSync(url);
+          const file = File({
+            url,
```

Comment:
File stamp accpes `uri` prop and not `url` prop. Addressed in https://github.com/swagger-api/apidom/pull/3569